warehouse/query-core/src/main/java/datawave/query/index/lookup/IndexInfo.java

```diff
@@ -339,203 +346,72 @@ public boolean intersect(List<JexlNode> delayedNodes) {
         return true;
     }
     
-    /**
```

Comment:
I Love that we are getting rid of all of this logic!

Reply:
same!